.github/service/model.py

```diff
@@ -0,0 +1,134 @@
+"""
+Models for Recommendations Service
+
+All of the models are stored in this module
+
+Models
+-------
+Recommendations - The recommendations resource is a representation a product recommendation based on another product.
+
+Atttibutes:
+-------
+product id (int) - a unique number which indicates a product
+related product id (int) - a unique number which indicates the recommended product of product A
+relationship type id (int) - the numbers which indicates the products' realationship: 0 - accessory, 1 - up-sells, 2 - cross-sells
+active status (boolean) - whether this recommendation pair is actived or not. 
+
+"""
+
+import logging
+from flask_sqlalchemy import SQLAlchemy
+
+# Create the SQLAlchemy object to be initialized later in init_db()
+db = SQLAlchemy()
+
+
+class DataValidationError(Exception):
+    """ Used for an data validation errors when deserializing """
+    pass
+
+class Recommendation(db.Model):
+    """
+    Class that represents a Recommendation
+
+    This version uses a relational database for persistence which is hidden
+    from us by SQLAlchemy's object relational mappings (ORM)
+    """
+
+    logger = logging.getLogger(__name__)
+    app = None
+
+    ##################################################
+    # Table Schema
+    ##################################################
+
+    id = db.Column(db.Integer, primary_key=True)
+    rel_id = db.Column(db.Integer, primary_key=True)
+    typeid = db.Column(db.Integer, primary_key = True)
```

Comment:
Do we need to set the `typeid` as primary key too? I think each `(id, rel_id)` will have only one corresponding type.